app/models/katello/sync_plan.rb

```diff
@@ -16,7 +16,7 @@ class SyncPlan < Katello::Model
 
     belongs_to :organization, :inverse_of => :sync_plans
     has_many :products, :class_name => "Katello::Product", :dependent => :nullify
-
+    belongs_to :foreman_tasks_recurring_logic, :inverse_of => :sync_plan, :class_name => "ForemanTasks::RecurringLogic", :dependent => :destroy
     validates_lengths_from_database
```

Comment:
mind adding this space between the associations and validations back?  for better readability.